staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject_test.go

```diff
@@ -412,7 +412,7 @@ func TestAddSubject(t *testing.T) {
 	for _, tt := range tests {
 		changed := false
 		var got []rbacv1.Subject
-		if changed, got = addSubjects(tt.existing, tt.subjects); (changed != false) != tt.wantChange {
+		if changed, got = addSubjects(tt.existing, tt.subjects); (changed) != tt.wantChange {
```

Comment:
I guess it you might as well change it from `(changed) != tt.wantChange` to `changed != tt.wantChange`, right?

Reply:
Yes. Have updated